meedan/check-web
# Fixing editing rules by replacing one-option menu by a straight edit button

## Description

Fixing editing rules by replacing one-option menu by a straight edit button.

Fixes CV2-4262.

## Type of change


## How has this been tested?

Integration tests were updated for this change. But it can be tested manually too:

- Create a rule 1
- Create a rule 2
- Click to edit rule 1: The form should show rule 1 content
- Go back
- Click to edit rule 2: The form should show rule 2 content

## Checklist
